.github/workflows/pipeline.yml

```diff
@@ -28,7 +28,7 @@ name: pipeline
 on:
   push:
     branches:
-      - main 
```

Comment:
I changed this file in previous commit (which I force pushed) which cases the whitespace cleanup here, I prefer to keep these file clean but strictly speaking its not related to this PR